lms/static/sass/discussion/elements/_navigation.scss

```diff
@@ -363,7 +363,7 @@
 
   &:hover,
   &:focus {
-    color: $link-color;
+    color: $forum-color-active-text;
```

Comment:
The background is ``palette(primary, base)`` so the foreground color now has to be a light color. The old contrast was very poor, but with the Pattern Library color changes it became invisible.

Note also that we have to use variables here because this partial is compiled into both Pattern Library and old school Sass files. We also should always use variables rather than direct calls to ``palette`` to make it simpler for theming to override.